CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 # Changelog
 
+## [9.5.0](https://github.com/theforeman/puppet-dns/tree/9.5.0) (2022-10-28)
+
+[Full Changelog](https://github.com/theforeman/puppet-dns/compare/9.4.0...9.5.0)
+
+**Implemented enhancements:**
+
+- stopped zone file from being created if zone type is forward [\#219](https://github.com/theforeman/puppet-dns/pull/219) ([ikonia](https://github.com/ikonia))
+- Remove dnssec-enable option for EL \>= 9 [\#218](https://github.com/theforeman/puppet-dns/pull/218) ([ikonia](https://github.com/ikonia))
+- Added defaultzonepath to template name.conf.erb [\#217](https://github.com/theforeman/puppet-dns/pull/217) ([benjamin-robertson](https://github.com/benjamin-robertson))
+
+**Merged pull requests:**
+
+- Change IRC support details for foreman website support URL [\#222](https://github.com/theforeman/puppet-dns/pull/222) ([ikonia](https://github.com/ikonia))
+- Use a better style in the test suite [\#220](https://github.com/theforeman/puppet-dns/pull/220) ([ekohl](https://github.com/ekohl))
```

Comment:
IMHO both are not changelog-worthy, but wouldn't block on that

Reply:
I can drop the latter but I'd like to keep the former in to give proper credit.

Reply:
Sure, as I said, this is nothing I'd block on, so edit as you see fit (or ignore me ;))